src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersContainerBuilderContext.cs

```diff
@@ -93,7 +93,10 @@ private void AddNullMemberInitializers(IMemberAssignmentMappingContainer contain
             if (!nullablePath.Member.CanSet)
                 continue;
 
-            if (!_initializedNullableTargetPaths.Add(nullablePath))
+            if (_initializedNullableTargetPaths.Contains((Mapping, nullablePath)))
+                continue;
```

Comment:
This additional contains call is not needed, is it?

Reply:
I just noticed that the first tuple value is different, and this totally makes sense here.